server/guiserver/apps.py

```diff
@@ -89,6 +89,10 @@ def server():
         'jujugui.sandbox': options.sandbox,
         'jujugui.raw': options.jujuguidebug,
         'jujugui.combine': (not options.jujuguidebug),
+        'jujugui.apiAddress': options.apiurl,
+        'jujugui.socketTemplate': '/environment/$uuid/api',
+        'jujugui.jujuCoreVersion': options.jujuversion,
+        'jujugui.uuid': options.uuid
```

Comment:
It's used as a means to pass the uuid of the initial environment to the gui so we can calculate the initial websocket.